### PR TITLE
Header

### PR DESCRIPTION
**Pull request by wildaces215**
Created nabber responsive but when I click the button the button come
out vertical. Will look into this when I get a chance

### CODE REVIEW

**Comment by Brentophillips** [comment]
Good work on the menu!

**Comment by wildaces215** [comment]
Two things

**Comment by roby-boy** [comment]
Mobile buttons align fixed (pushed)

File: beta_files/css/main.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.navbar-brand {
+  padding: 0px;
+  height:70px; /* firefox bug fix */
```
Comment (fbucchi-gpsw) [inline beta_files/css/main.css]: You can remove the comment `/* firefox bug fix */` because it's valid also for `chrome`

Comment (fbucchi-gpsw) [inline beta_files/css/main.css]: Alphabetize

File: beta_files/css/main.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.navbar-brand>img {
+  height: 100%;
+  padding: 15px; /* firefox bug fix */
+  width: auto;
+}
+.navbar-btn{
+    width:50px;
+    height:50px;
+    margin:10px;
+}
```
Comment (fbucchi-gpsw) [inline beta_files/css/main.css]: - Indent 2 spaces
- Alphabetize

File: beta_files/css/main.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@media screen and (max-width:768px){
+    .navbar-nav{
+        display:inline;
+    }
```
Comment (fbucchi-gpsw) [inline beta_files/css/main.css]: I think you can remove this

File: beta_files/css/main.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    .nav > li {
+        float: left;
+    }
```
Comment (fbucchi-gpsw) [inline beta_files/css/main.css]: Indent 2 spaces

**Comment by fbucchi-gpsw** [comment]
@wildaces215  @roby-boy  I added some comments

**Comment by fbucchi-gpsw** [comment]
@Brentophillips we didn't address my comments..

**Pull request merged**